RussianFilter/sections/css_extended.txt

```diff
@@ -238,6 +238,7 @@ nastup.net#?#.l-content > section.color_custom:has(> div > div.offset_small > di
 softportal.com#?#.relap-default-wrapper div[data-relap-max-column-quantity] > div.js-relap__item:has(> div.js-relap__item-inner > a > div.relap-default__item__label:contains(Реклама))
 overclockers.ru#?#div[itemprop="articleBody"] > h4:contains(реклама)
 vplate.ru#?#.article > div[style]:has(> ins.adsbygoogle)
+eda.ru#?#[style]:has(+ .js-advertizing__observer)
```

Comment:
check it's performance plz

also looks too wide, what if it causes false positives?

Reply:
I suggest to use such rules
`eda.ru##body > [style*="float"][style*="width"]:not([class]):not([id])`
`eda.ru##section.main-content > [style*="float"][style*="width"][style*="flex-direction"]:not([id])`

Reply:
too many `:not()`